test/statetest/statetest.cpp

```diff
@@ -80,18 +81,21 @@ int main(int argc, char* argv[])
             ->required()
             ->check(CLI::ExistingPath);
 
-        bool trace_flag = false;
-        app.add_flag("--trace", trace_flag, "Enable EVM tracing");
+        bool trace = false;
```

Comment:
I think the name `tracing_enabled` would be better here.

Reply:
Same applies to `trace_summary`

Reply:
I'm not sure this is good idea to have variable name different than CLI option.